cheroot/workers/threadpool.py

```diff
@@ -92,6 +93,53 @@ def __init__(self, server):
         }
         threading.Thread.__init__(self)
 
+    def log_start_stats(self):
+        if self.server.stats['Enabled']:
+            self.start_time = time.time()
+
+    def log_close_stats(self):
+        if self.server.stats['Enabled']:
+            self.requests_seen += self.conn.requests_seen
+            self.bytes_read += self.conn.rfile.bytes_read
+            self.bytes_written += self.conn.wfile.bytes_written
+            self.work_time += time.time() - self.start_time
+            self.start_time = None
+
+    def conn_expired(self, conn, last_active, cur_time):
+        srv_timeout = self.server.timeout
+        return cur_time - last_active > srv_timeout
+
+    def get_expired_conns(self, conn_socks, cur_time):
+        for conn, last_active in tuple(conn_socks.values()):
+            if self.conn_expired(conn, last_active, cur_time):
+                yield conn
+
+    def close_conns(self, conn_list, conn_socks):
+        for conn in conn_list:
+            conn.communicate()  # allow for 408 to be sent
+            conn.close()
+            conn_socks.pop(conn.socket)
+
+    def process_conns(self, conn_socks):
+        rlist = []
+        socks = [sck for sck in conn_socks.keys() if sck.fileno() > -1]
+        if socks:
+            rlist = select.select(socks, [], [], 0)[0]
```

Comment:
1) why process only first item?
2) what `rlist` stands for? I suggest you use actual words + snake_case for vars